soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -209,12 +226,57 @@ void func_809DF730(EnCow* this, GlobalContext* globalCtx) {
     }
 }
 
+bool EnCow_IsSecondCowInTower(EnCow* this, GlobalContext* globalCtx) {
+    return 
+        gSaveContext.n64ddFlag && 
+        globalCtx->sceneNum == SCENE_SOUKO && 
+        (this->actor.world.pos.x + this->actor.world.pos.z == -173 || this->actor.world.pos.x + this->actor.world.pos.z == -72);
```

Comment:
using x+z seems really odd here, is there a reason to do that instead of just checking against known x,y,z coords?

Reply:
As said a little bit below here, this should also check for the actor params instead of position values.